processing-scripts/01-filtering-data/02-process-filter-nbi-bridges.py

```diff
@@ -84,49 +37,34 @@ def convert_to_gpkg(df, output_gpkg_file):
     """
 
     # Create geometry from latitude and longitude
-    geometry = [Point(xy) for xy in zip(df["LONG_Final"], df["LAT_Final"])]
+    geometry = [
+        Point(xy)
+        for xy in zip(df["17 - Longitude (decimal)"], df["16 - Latitude (decimal)"])
+    ]
     gdf = gpd.GeoDataFrame(df, geometry=geometry)
 
     gdf.to_file(output_gpkg_file, driver="GPKG")
 
     print(f"GeoPackage saved successfully to {output_gpkg_file}")
 
 
-def process_coordinates(
-    input_csv, output_convert_csv, output_duplicate_exclude_csv, output_gpkg_file
-):
+def process_coordinates(input_csv, output_duplicate_exclude_csv, output_gpkg_file):
     """
     Funtion to perform processing of coordinates and filtering of bridges
     """
-    # Read the input CSV
-    df = pd.read_csv(
-        input_csv, dtype={"LAT_016": str, "LONG_017": str}, low_memory=False
-    )
-
-    # Handle missing values by filling with zeros
-    df["LAT_016"] = df["LAT_016"].fillna("00000000")
-    df["LONG_017"] = df["LONG_017"].fillna("000000000")
-
-    # Calculate LAT_DD and LONG_DD
-    df["LAT_DD_new"] = df["LAT_016"].apply(calculate_lat_dd)
-    df["LONG_DD_new"] = df["LONG_017"].apply(calculate_long_dd)
-
-    # Write to a new CSV
-    df.to_csv(output_convert_csv, index=False)
 
+    df = pd.read_csv(input_csv)
     # Exclude duplicate bridges and save the result to a CSV
     df = exclude_duplicate_bridges(df, output_duplicate_exclude_csv)
 
     # Convert the final DataFrame to a GeoPackage file
     convert_to_gpkg(df, output_gpkg_file)
 
 
-input_csv = "input-data/Kentucky-NBI-bridge-data.csv"
-output_convert_csv = "output-data/csv-files/Kentucky-bridge-converted-coordinates.csv"
+# input_csv = "input-data/Kentucky-NBI-bridge-data.csv"
+input_csv = "input-data/NBI-Kentucky-bridge-data.csv"
 output_duplicate_exclude_csv = (
     "output-data/csv-files/Kentucky-bridge-chosen-coordinates.csv"
 )
 output_gpkg_file = "output-data/gpkg-files/NBI-Kentucky-Bridge-Data.gpkg"
-process_coordinates(
-    input_csv, output_convert_csv, output_duplicate_exclude_csv, output_gpkg_file
-)
+process_coordinates(input_csv, output_duplicate_exclude_csv, output_gpkg_file)
```

Comment:
@varun-andhra-mapup do we have AI code review added to this? If yes, where are the AI suggestions? Also, you need to make a change to start the code review process

Reply:
@maneesh-mapup We have the PR AI Agent installed in this repo. However, this PR is 2 weeks old and has already been closed and merged into main. I am creating a new PR now with some logging changes, where the PR Agent should run by itself.